envs/env_base.py

```diff
@@ -4,9 +4,16 @@ class EnvironmentBase:
     Args:
         player1, player2: the player objects.
     """
+
     def __init__(self, player1, player2):
-        self.player1 = player1
-        self.player2 = player2
+        self.player1 = {
```

Comment:
This is implementation detail - some environments may not use a `symbol` to determine what to put into the board, if all there is a board.

Reply:
done.